recipes/sherloc/inference/sherloc.py

```
@@ -0,0 +1,1034 @@
+                        ) or "is longer than the model's context length" in str(e):
+                            log_warning(
+                                "SHERLOC generation failed due to running out of context. "
+                                "Failing for subsequent subtasks automatically.",
+                                indent=16,
+                            )
+                            status = "failed"
+                            reason = "context_length_exceeded"
+                            break
+                        log_warning(f"SHERLOC generation failed with BadRequestError: {e}", indent=12)
+                        status = "failed"
+                        reason = f"bad_request_error: {str(e)}"
```

Ruff (0.16.2)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)